meta/bump

```diff
@@ -26,7 +26,7 @@ set -e
 git pull
 
 # Update the formula and the version file with the new version
-sed -i '' "s/VERSION=.*/VERSION=\"$VERSION\""/ "$ROOT_DIR/formula/ih-core.rb"
```

Comment:
This extra arg made this not work on Linux. Does that work on MacOS?

Reply:
Yes, MacOS has some non-posix `sed`.